boms/integration-tests/src/test/java/com/google/cloud/MaximumLinkageErrorsTest.java

```diff
@@ -44,9 +45,22 @@
   public void textMaximumLinkageErrors() 
       throws IOException, MavenRepositoryException, RepositoryException {
 
+    Bom baseline = RepositoryUtility.readBom("com.google.cloud:libraries-bom:1.2.0");
+
     Path bomFile = Paths.get("../cloud-oss-bom/pom.xml");
     Bom bom = RepositoryUtility.readBom(bomFile);
     
+    ImmutableSetMultimap<SymbolProblem, ClassFile> oldProblems = findProblems(baseline);
+    ImmutableSetMultimap<SymbolProblem, ClassFile> newProblems = findProblems(bom);
+
+    Assert.assertTrue("New linkage errors introduced", newProblems.keys().size() <= 525);
+    
+    // Check that no new linkage errors have been introduced since 1.2.0
+    Truth.assertThat(oldProblems).containsAtLeastEntriesIn(newProblems);
+  }
+
+  private static ImmutableSetMultimap<SymbolProblem, ClassFile> findProblems(Bom bom)
```

Comment:
As discussed earlier, I'm thinking this usage to be `LinkageChecker.create(Bom bom)` to run linakge check for a BOM. Then the logic can be shared by dashboard, linakge-monitor, and this test.